code/__DEFINES/traits.dm

```diff
@@ -111,7 +111,10 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NOBREATH			"no_breath"
 #define TRAIT_ANTIMAGIC			"anti_magic"
 #define TRAIT_HOLY				"holy"
-#define TRAIT_DEPRESSION		"depression"
```

Comment:
You dont use the new traits for anything, only the mood ones. Unless you got a good reason, undo it.